<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="MainBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>631</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:none;
border-radius:8px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="AirQualityLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>481</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>300</pointsize>
     </font>
    </property>
    <property name="text">
     <string>25</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>室内空气质量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>610</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AirQuality</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="XProgressBar" name="Temp_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>710</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="XProgressBar" name="Humi_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>670</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>710</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ModePage">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>840</y>
     <width>911</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#ModePage{
background-color:white;
border:none;
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Solid">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>15</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">max-width:120px;
 max-height:120px;
 min-width:120px;
 min-height:120px;
border-radius:60px;
background-color: white;
 border-style: solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/Wind/Solid_OFF.png</normaloff>:/Wind/Solid_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Nature">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>15</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">max-width:120px;
 max-height:120px;
 min-width:120px;
 min-height:120px;
border-radius:60px;
background-color: white;
 border-style: solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/Wind/Natural_OFF.png</normaloff>:/Wind/Natural_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Surround">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>15</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">max-width:120px;
 max-height:120px;
 min-width:120px;
 min-height:120px;
border-radius:60px;
background-color: white;
 border-style: solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/Wind/Surround_OFF.png</normaloff>:/Wind/Surround_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Sea">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>15</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">max-width:120px;
 max-height:120px;
 min-width:120px;
 min-height:120px;
border-radius:60px;
background-color: white;
 border-style: solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/Wind/Sea_OFF.png</normaloff>:/Wind/Sea_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FanPage">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>600</y>
     <width>911</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#FanPage{
background-color:white;
border:none;
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border:none
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Low">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>15</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">max-width:120px;
 max-height:120px;
 min-width:120px;
 min-height:120px;
border-radius:60px;
background-color: white;
 border-style: solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/Wind/Low_OFF.png</normaloff>:/Wind/Low_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="High">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>15</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">max-width:120px;
 max-height:120px;
 min-width:120px;
 min-height:120px;
border-radius:60px;
background-color: white;
 border-style: solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/Wind/High_OFF.png</normaloff>:/Wind/High_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Mid">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>15</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">max-width:120px;
 max-height:120px;
 min-width:120px;
 min-height:120px;
border-radius:60px;
background-color: white;
 border-style: solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/Wind/Mid_OFF.png</normaloff>:/Wind/Mid_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="OutDoorAir">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>330</y>
     <width>811</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#OutDoorAir{
background-color:white;
border:none;
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Pm10P">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PM10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Pm25P">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PM2.5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TVCOP">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TVCO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Co2P">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CO2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="HchoP">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>甲醛</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="XProgressBar" name="InsidePm10_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="InsidePm25_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="InsideTvco_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="InsideCo2_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="InsideHcho_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="InDoorBox">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>60</y>
     <width>51</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:none;
border-radius:8px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="OutDoorBox">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>330</y>
     <width>51</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:none;
border-radius:8px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="InDoorAir">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>60</y>
     <width>811</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#InDoorAir{
background-color:white;
border:none;
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="OutSideNo2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OutSidePm25">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PM2.5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OutSidePm10">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PM10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OutSideSo2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>So2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="XProgressBar" name="OutSideNo2_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="OutSidePm10_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="OutSideSo2_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="OutSidePm25_Xpro" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="AiMode">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>880</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:none;
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>1000</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AI模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XProgressBar</class>
   <extends>QWidget</extends>
   <header>xprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../other/main.qrc"/>
 </resources>
 <connections/>
</ui>
